src/snc/agents/steady_state_agents/steady_state_policy_agent.py

```diff
@@ -81,7 +81,12 @@ def map_state_to_actions(self, state: types.StateSpace, **override_args: Dict) \
 
         # Obtain physically feasible actions from MPC policy.
         actions = self.mpc_policy.obtain_actions(
-            state=state, mpc_variables=self.mpc_variables,
+            state=state,
+            x_star = state,
+            x_eff = state,
+            r_idling_set = np.array([]),
+            draining_resources = set(),
```

Comment:
@sergiovalmac 

simulation to estimate asymptotic covariance uses the same mpc policy as the real agent but passes a different fluid policy. In case of Fox instead of fluid policy, effective state should be passed to mpc policy. how can we work around it here?